### DISCUSSION TITLE
西瓜撞地球

### DISCUSSION THREAD

**Discussion by ix2213** [MaLuns/hexo-theme-async#51] [Show 分享展示]
name: 西瓜撞地球
url: https://www.bio-w.cn
avatar:  https://bu.dusays.com/2022/10/17/634d313602a1e.png
desc:  我在人间贩卖彩虹

**Comment by ix2213** [comment]
有个问题 升级失败   捣鼓了一晚上都是v1.1.1

**Reply by MaLuns** [reply]
因为使用 git tag 来控制发布 npm 版本的，所以不会去更新 package.json 里 version 字段，

如果你是将文件拉取到本地的方式，默认情况下版本是不会变的，后面会考虑解决这个问题
